app/themeToggle.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from 'react';
+import sunIcon from "../public/sun.svg";
+import moonIcon from "../public/moon.svg";
```

Comment:
reminder to use consistent quotations


```suggestion
import { useState, useEffect } from 'react';
import sunIcon from '../public/sun.svg';
import moonIcon from '../public/moon.svg';
```